docs/running-locally.md

```diff
@@ -101,6 +101,17 @@ If you made changes to the executor, you need to build the image:
 make argoexec-image
 ```
 
+Or you can use parameters to compile images that support specific platforms or multiple platforms with `TARGET_PLATFORM`:
+
+```bash
+# only support arm64
+make argoexec-image TARGET_PLATFORM=linux/arm64
+# only support amd64
+make argoexec-image TARGET_PLATFORM=linux/amd64
+# support arm64 and amd64
```

Comment:
```suggestion
# compile for both arm64 and amd64
```

- "compile", not "support"
- I think the last example is sufficient for all purposes, as it should be intuitive that you can remove the comma if you only need one, IMO. unless you think otherwise @Joibel

Reply:
yeah, thanks for your comment, i have fix it, could you have a look again ? @agilgur5

Reply:
For future reference, you can use the "re-request a review" button on GH, then it also pops up in my notifications as a review request specifically